impeller/typographer/typographer_unittests.cc

```diff
@@ -325,46 +325,6 @@ TEST_P(TypographerTest, RectanglePackerAddsNonoverlapingRectangles) {
   ASSERT_EQ(packer->PercentFull(), 0);
 }
 
-TEST_P(TypographerTest, GlyphAtlasTextureIsRecycledWhenContentsAreRecreated) {
```

Comment:
This test really depends on the exact atlas sizing and how efficient the packing is, making it unstable. I'm removing it for now.